docs/source/intro.rst

```diff
@@ -2,189 +2,145 @@
 Introduction
 ============
 
-The `conda` command is the primary interface for managing installations
-of various packages.  It can query and search the package index and current
-installation, create new environments, and install and update packages
-into existing conda environments.
+Conda is a cross-platform, Python-agnostic binary package manager. Conda can:
+
+- `Search <commands/conda-search.html>`_ for packages in a repository/package
+  index/channel or a current installation.
+- `Create <commands/conda-create.html>`_ conda environments.
+- `Install <commands/conda-install>`_ and `update <conda-update.html>`_
+  packages into new and existing environments.
 
-------------------
-Conda Overview
-------------------
 
 .. _package:
 .. index::
     pair: terminology; package
 
-.. _environment:
-.. index::
-    pair: terminology; environment
-
-conda is an application for finding and installing software packages.
-A conda `package` is a binary tarball containing system-level libraries,
-Python modules, executable programs, or other components.
-conda keeps track of dependencies between packages and platform
-specifics, making it simple to create working environments from different
-sets of packages.
 
-A `conda environment` is a filesystem directory that contains a specific
-collection of conda packages.  As a concrete example, you might want to
-have one environment that provides NumPy 1.7, and another environment that
-provides NumPy 1.6 for legacy testing.  conda makes this kind of mixing
-and matching easy.  To begin using an environment, simply set
-your **PATH** variable to point to its `bin` directory.
+Packages
+--------
 
-.. _channel:
-.. index::
-    pair: terminology; channel
+A conda `package` is a binary archive (tarball) containing system-level
+libraries, Python modules, executable programs, or other components. Conda
+keeps track of dependencies between packages and platform specifics, making it
+simple to install sets of packages.
 
-.. _locally_available:
-.. index::
-    pair: terminology; locally available
+A typical package file name could be something like:
 
-.. _activated:
-.. index::
-    pair: terminology; activated
+    ``gevent-websocket-0.3.6-py27_0.tar.bz2``
 
-.. _deactivated:
-.. index::
-    pair: terminology; deactivated
+    <package-name>-<version>-<build_string>.tar.bz2
```

Comment:
Agree but for intro it felt overkill to me. Or should we remove any reference to a filename at this point?